action-server/tests/lib/test_persistence.py

```diff
@@ -0,0 +1,165 @@
+from typing import Any, Dict
+from unittest import TestCase
+from unittest.mock import call, patch
+
+from actions.lib.persistence import (
+    METADATA_SLOT,
+    REMINDER_ID_METADATA_PROPERTY,
+    cancel_reminder,
+    save_reminder,
+    store_assessment,
+)
+from db.assessment import (
+    FEEL_WORSE_SLOT,
+    HAS_COUGH_SLOT,
+    HAS_DIFF_BREATHING_SLOT,
+    HAS_FEVER_SLOT,
+    SYMPTOMS_SLOT,
+    Assessment,
+)
+from db.reminder import PHONE_NUMBER_SLOT, Reminder
+
+HASHIDS_SALT = "abcd1234"
+HASHIDS_MIN_LENGTH = 4
+ENV = {
+    "REMINDER_ID_HASHIDS_SALT": HASHIDS_SALT,
+    "REMINDER_ID_HASHIDS_MIN_LENGTH": str(HASHIDS_MIN_LENGTH),
+}
+
+REMINDER_ID = 1
+HASHED_ID = "ZaeZ"
+
+DEFAULT_ASSESSMENT_SLOT_VALUES: Dict[str, Any] = {
+    SYMPTOMS_SLOT: "moderate",
+    FEEL_WORSE_SLOT: True,
+    HAS_COUGH_SLOT: True,
+    HAS_DIFF_BREATHING_SLOT: True,
+    HAS_FEVER_SLOT: True,
+}
+
+DEFAULT_SLOTS_VALUES: Dict[str, Any] = {
+    METADATA_SLOT: {REMINDER_ID_METADATA_PROPERTY: HASHED_ID},
+    **DEFAULT_ASSESSMENT_SLOT_VALUES,
+    "ignored_solt": "Boba Fett",
```

Comment:
```suggestion
    "ignored_slot": "Boba Fett",
```

Reply:
this is kind of ironic since this slot is ignored - will fix it in another branch to merge now that I can.